### ISSUE TITLE
Evaluation Code Release

### ISSUE THREAD

**Issue by YuanxunLu** [jasonyzhang/RayDiffusion#10]
Hi, thanks for your impressive work! 

I'm wondering when the evaluation code would be released? or could you kindly provide some reference codes/repos for pose evaluation?

Hope to hear from you and thanks for your work!

**Comment by jasonyzhang** [comment]
Hi, due to significant amount of travel, the evaluation and training code probably will not be public until around ICLR conference time. In the meantime, the evaluation code is essentially the same as [here](https://github.com/amyxlase/relpose-plus-plus). However, you should use the co3d dataloader provided in this repo (the image normalization parameters are slightly different)

**Comment by jasonyzhang** [comment]
Evaluation code is now available!
